apps/web/src/pages/templates/components/notificationTemplateSchema.ts

```diff
@@ -89,95 +165,177 @@ export const schema = z
+                            DaysEnum.MONDAY,
+                            DaysEnum.TUESDAY,
+                            DaysEnum.WEDNESDAY,
+                            DaysEnum.THURSDAY,
+                            DaysEnum.FRIDAY,
+                            DaysEnum.SATURDAY,
+                            DaysEnum.SUNDAY,
+                          ])
+                        ),
+                      })
+                      .passthrough()
+                      .optional(),
+                    [DigestUnitEnum.MONTHS]: z
+                      .object({
+                        amount: z.string(),
+                        atTime: z.string(),
+                        monthDays: z.array(z.number()),
+                        monthlyType: z.enum([MonthlyTypeEnum.EACH, MonthlyTypeEnum.ON]),
+                        ordinal: z
+                          .enum([
+                            OrdinalEnum.FIRST,
+                            OrdinalEnum.SECOND,
+                            OrdinalEnum.THIRD,
+                            OrdinalEnum.FOURTH,
+                            OrdinalEnum.FIFTH,
+                            OrdinalEnum.LAST,
+                          ])
+                          .optional(),
+                        ordinalValue: z
+                          .enum([
+                            OrdinalValueEnum.DAY,
+                            OrdinalValueEnum.WEEKDAY,
+                            OrdinalValueEnum.WEEKEND,
+                            OrdinalValueEnum.MONDAY,
+                            OrdinalValueEnum.TUESDAY,
+                            OrdinalValueEnum.WEDNESDAY,
+                            OrdinalValueEnum.THURSDAY,
+                            OrdinalValueEnum.FRIDAY,
+                            OrdinalValueEnum.SATURDAY,
+                            OrdinalValueEnum.SUNDAY,
+                          ])
+                          .optional(),
+                      })
+                      .passthrough()
+                      .optional(),
+                  })
+                  .passthrough()
+                  .optional(),
+              })
+              .passthrough()
+              .optional(),
+            delayMetadata: z
+              .object({
+                type: z.enum([DelayTypeEnum.REGULAR, DelayTypeEnum.SCHEDULED]),
+                [DelayTypeEnum.REGULAR]: z
+                  .object({
+                    amount: z.string(),
+                    unit: z.string(),
+                  })
+                  .passthrough()
+                  .optional(),
+                [DelayTypeEnum.SCHEDULED]: z
+                  .object({
+                    delayPath: z.string(),
+                  })
+                  .passthrough()
+                  .optional(),
               })
               .passthrough()
               .optional(),
           })
           .passthrough()
-          .superRefine((step: any, ctx) => {
+          .superRefine((step, ctx) => {
```

Comment:
simplified the validation process... we don't validate any digest fields like amounts, at time, as they should have default value set on blur when empty